modules/i3s/src/lib/parsers/parse-i3s-webscene.ts

```diff
@@ -0,0 +1,46 @@
+import type {I3SWebScene, I3SWebSceneOperationalLayer} from '../../types';
+
+const ARCGIS_SCENE_SERVER_LAYER_TYPE = 'ArcGISSceneServiceLayer';
```

Comment:
Is this an ARCGIS specific loader? If so should the name reflect that? Similar to Cesium ION loader?

Reply:
Renamed it to `ArcGisWebSceneLoader`.